doc/source/contributing.rst

```diff
@@ -60,46 +54,46 @@ encouraged to install this tool via:
 
 Tox
 ~~~
-Tests can be run using `tox`_. The project defines the tox environments in ``tox.ini``.
-The following tox environments are provided:
+Tests can be run using `tox`_. The project defines the tox environments in the ``tox.ini``
+file. The following tox environments are provided:
 
 .. vale off
 
-- ``tox -e style``: checks for coding style quality.
-- ``tox -e tests``: runs all tests and checks code coverage (see :ref:`ref_serveraccess` for requirements).
-- ``tox -e doc``: checks the documentation building process.
+- ``tox -e style``: Checks for coding style quality.
+- ``tox -e tests``: Runs all tests and checks code coverage. (For requirements, see :ref:`ref_serveraccess`.)
+- ``tox -e doc``: Checks the documentation-building process.
 
 .. vale on
 
-Optionally add the ``-- -m "not integration"`` suffix to the commands above to skip integration
-tests. For example, ``tox -e tests -- -m "not integration"`` will only run tests that
-do not require a Granta MI instance.
+Optionally, add the ``-- -m "not integration"`` suffix to the commands above to skip integration
+tests. For example, this command only runs tests that do not require a Granta MI instance::
+
+     tox -e tests -- -m "not integration"
 
 .. _ref_serveraccess:
 
 Server access
 --------------
 
-Running integration tests and building the examples requires access to a valid Granta MI instance
-(see :ref:`ref_software_requirements`).
+As indicated in :ref:`ref_software_requirements`, running integration tests and building the examples
+requires access to a valid Granta MI instance.
 
-External contributors may not have an instance of Granta MI at their disposal. Prior to creating a pull request with the
-desired changes, they should make sure that unit tests pass (:ref:`ref_tox`), static code validation and
-styling pass (:ref:`pre-commit <ref_precommit>`), and that the documentation can be generated successfully without the
-examples (:ref:`Documenting <ref_documenting>`).
+External contributors may not have an instance of Granta MI at their disposal. Prior to creating a
```

Comment:
```suggestion
External contributors might not have an instance of Granta MI at their disposal. Prior to creating a
```
Google dev doc style guide says to reserve "may" for situations where permission is granted.